src/main/java/com/javadiscord/jdi/internal/api/impl/scheduledevents/CreateScheduledEventRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.javadiscord.jdi.internal.api.impl.scheduledevents;
+
+import com.javadiscord.jdi.internal.api.DiscordRequest;
+import com.javadiscord.jdi.internal.api.DiscordRequestBuilder;
+import com.javadiscord.jdi.internal.models.guild.EntityMetadata;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public record CreateScheduledEventRequest(
+        String guildId,
+        Optional<String> channelId,
+        Optional<EntityMetadata> entityMetadata,
+        String name,
+        int privacyLevel, // 2 is only option atm
+        long scheduledStartTime, // ISO8601 timestamp (assuming long) note to self to check back
```

Comment:
@Taz03 can you provide the correct format for `scheduledStartTime`

Reply:
OffsetDateTime

Reply:
Done!